src/amo/components/SearchContextCard/index.js

```diff
@@ -93,6 +94,47 @@ export class SearchContextCardBase extends React.Component<InternalProps> {
             );
           }
           break;
+        case ADDON_TYPE_OPENSEARCH:
+          if (categoryName) {
```

Comment:
since 'search' doesn't have any categories, I believe you can remove this check here:

```diff
--- a/src/amo/components/SearchContextCard/index.js
+++ b/src/amo/components/SearchContextCard/index.js
@@ -95,27 +95,7 @@ export class SearchContextCardBase extends React.Component<InternalProps> {
           }
           break;
         case ADDON_TYPE_OPENSEARCH:
-          if (categoryName) {
-            if (query) {
-              searchText = i18n.sprintf(
-                i18n.ngettext(
-                  '%(count)s Search Tool found for "%(query)s" in %(categoryName)s',
-                  '%(count)s Search Tools found for "%(query)s" in %(categoryName)s',
-                  count,
-                ),
-                { count: i18n.formatNumber(count), query, categoryName },
-              );
-            } else {
-              searchText = i18n.sprintf(
-                i18n.ngettext(
-                  '%(count)s Search Tool found in %(categoryName)s',
-                  '%(count)s Search Tools found in %(categoryName)s',
-                  count,
-                ),
-                { count: i18n.formatNumber(count), categoryName },
-              );
-            }
-          } else if (query) {
+          if (query) {